src/odemis/driver/test/xt_client_test.py

```diff
@@ -712,6 +711,72 @@ def test_apply_autofocus(self):
         autofocus_state = self.microscope.is_autofocusing(channel)
         self.assertEqual(autofocus_state, False)
 
+    def test_get_mpp_orientation(self):
+        """
+        Simple test, checks if no error occur during calling the method and the return value.
+        """
+        mpp_orientation = self.microscope.get_mpp_orientation()
+        self.assertIsInstance(mpp_orientation, float)
+        # TODO K.K. check range with Wilco
+        self.assertTrue(0 < mpp_orientation < 360)
+
+    def test_get_aperture_index(self):
+        """
+        Simple test, checks if no error occur during calling the method and the return value.
+        """
+        aperture_index = self.microscope.get_aperture_index()
+        self.assertIsInstance(aperture_index, float)
+        self.assertTrue(0 < aperture_index < 14)
+
+    def test_set_aperture_index(self):
+        """
+        Simple test, checks if no error occur during calling the method.
+        """
+        # TODO Check if setting of the value actually happens and if an int value is also allowed.
+        self.microscope.set_aperture_index(10)
+        aperture_index = self.microscope.get_aperture_index()
+        self.assertEqual(aperture_index, 10)
+
+    def test_aperture_index_info(self):
+        """
+        Simple test, checks if no error occur during calling the method and the return value.
+        """
+        aperture_index_info = self.microscope.aperture_index_info()
+        self.assertIsInstance(aperture_index_info, dict)
+        self.assertIsInstance(aperture_index_info["range"], list)
+        self.assertEqual(len(aperture_index_info["range"]), 2)
+
+    def test_get_beamlet_index(self):
+        """
+        Simple test on checks if no error occur during calling the method and the return value.
+        """
+        beamlet_index = self.microscope.get_beamlet_index()
+        self.assertIsInstance(beamlet_index, tuple)
+        self.assertEqual(len(beamlet_index), 2)
+        self.assertTrue(1 < beamlet_index[0] < 8)
+        self.assertTrue(1 < beamlet_index[1] < 8)
+
+    def test_set_beamlet_index(self):
+        """
+        Simple test on checks if no error occur during calling the method and the return value.
+        """
+        # TODO Check if setting of the value actually happens and if an int value is also allowed.
+        self.microscope.set_beamlet_index((4, 4))
+        beamlet_index = self.microscope.get_beamlet_index()
+        self.assertEqual(beamlet_index, (4, 4))
```

Comment:
Also here I suggest you do it the same way as the other tests: 

- start with getting the currently set value
- then change to a different value and check it is set
- then put back the originally set value